front/src/common/Map/Settings/MapSettingsSignals.js

```diff
@@ -16,13 +16,38 @@ export default function MapSettingsSignals() {
     lights: lightsIcon,
   };
 
+  const switchAllSettings = (settingSwitch, stateOfSettings) => {
+    const allSettings = {};
+
+    CONSTS_SETTINGS.forEach(
+      (settingKey) => (allSettings[settingKey] = !stateOfSettings[settingSwitch])
```

Comment:
written like this, it seems like you want to use the value returned.

Here we just want to use the side-effect (adding a key to the object). 

you can simply do this :

```
    CONSTS_SETTINGS.forEach(
      (settingKey) => {
         allSettings[settingKey] = !stateOfSettings[settingSwitch]
      }
   );
```